webui/react/src/components/TaskActionDropdown.tsx

```diff
@@ -159,7 +167,9 @@ const TaskActionDropdown: React.FC<Props> = ({ task, onComplete, curUser }: Prop
   }
 
   if (menuItems.length === 0) {
-    return (
+    return children ? (
+      <>{children}</>
```

Comment:
nit: i don't think wrapping children in a fragment here is necessary

Reply:
The fragment-wrapping was to placate `tsc`,  which before was saying:
```
Type '({ task, onComplete, curUser, onVisibleChange, children, }: PropsWithChildren<Props>) => {}' 
is not assignable to type 'FC<Props>'.
  Type '{}' is missing the following properties from type 'ReactElement<any, any>': 
type, props, key  TS2322
```

Looks like the below is also allowed. Is this better?
```typescript
return (children as JSX.Element) ??
  <div className={css.base} title="No actions available" onClick={stopPropagation}>
   <button disabled>
      <Icon name="overflow-vertical" />
    </button>
  </div>
```

Reply:
Yeah, I think I like that way better.